finetune/encoding/input_encoder.py

```diff
@@ -8,9 +8,15 @@
 import spacy
 import numpy as np
 
+NLP = None
+
+def get_spacy():
```

Comment:
What's the reasoning behind this change?

Reply:
It does actually make mocking this out a lot easier for customv1 though--I think I'll copy this formulation over there :)

Reply:
Mostly just means we don't load spacy at import or if we don't use it.